repotracker/repotracker.go

```diff
@@ -765,7 +765,10 @@ func createVersionItems(ctx context.Context, v *model.Version, ref *model.Projec
 			var lastActivated *model.Version
 			lastActivated, err = model.VersionFindOne(model.VersionByLastVariantActivation(ref.Identifier, buildvariant.Name))
 			if err != nil {
-				return errors.Wrap(err, "problem getting activatation time for variant")
+				grip.Error(message.WrapError(err, message.Fields{
+					"message": "error finding last activated",
+					"version": v.Id,
+				}))
```

Comment:
Did you mean to add a `continue` here as well?

Reply:
Nope, I want it to fall through to using the default value

Reply:
I think though that that means that an error from the database would mean automatically activating the variant.

Reply:
It would, but presumably if there are db problems we wouldn't get here in the first place. On the off-chance that only this read fails, I feel like it's fine to insert and activate the version. More likely, this will prevent bugs in that query from destroying the repotracker